examples/wave-lazy.py

```diff
@@ -0,0 +1,131 @@
+"""Demonstrate wave-lazy serial example."""
```

Comment:
Should we just make this a part of the `wave-eager.py` and do the switch via a command line argument? Avoids duplication.

Reply:
:+1: (And rename to `wave.py`?)

Reply:
Done.